### PR TITLE
refactor: make `getDefaultPair` a pure function & rename

### PR DESCRIPTION
**Pull request by 0xApotheosis**
## Description

In paving the way for a `tradeRoutes` refactor, this PR makes the `getDefaultPair` callback a pure function.

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

Contributes to https://github.com/shapeshift/web/issues/2556

## Risk

Small, no logic changes.

### Engineering

If default assets still load when connecting to a wallet, loading a new page etc we are 🛍️ 

### Operations

If default assets still load when connecting to a wallet, loading a new page etc we are 🛍️ 

## Screenshots (if applicable)

N/A

### CODE REVIEW

**Comment by 0xApotheosis** [comment]
Current dependencies on/for this PR:
* 08-31-refactor_simplify_useToggle
  * **PR #2618** <a href="https://app.graphite.dev/github/pr/shapeshift/web/2618" target="_blank">Graphite</a>  👈
    * **PR #2619** <a href="https://app.graphite.dev/github/pr/shapeshift/web/2619" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/2618?utm_source=stack-comment).

**Review by gomesalexandre — APPROVED** [review]
Tested:

- Connecting a wallet (Native/MM), spotted no regressions from develop, default pairs are still loaded in dashboard/assets page
- Reconnecting a wallet(Native/MM), default pairs do not load in dashboard but this isn't a regression from develop

As far as refactoring this goes, does what it says on the box

Added via Giphy

**Comment by 0xApotheosis** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/2618?utm_source=stack-comment).


Job Reference: KFwa69VVfFg5xHjr8un0

**Pull request merged**
